zigpy/appdb.py

```diff
@@ -216,6 +216,11 @@ def load(self):
                 if cluster in ep.in_clusters:
                     clus = ep.in_clusters[cluster]
                     clus._attr_cache[attrid] = value
+                    LOGGER.debug("Attribute id: %s value: %s", attrid, value)
+                    if cluster == 0 and attrid == 4:
```

Comment:
Can you please use the cluster_id from the cluster class, so it is easier to understand.